Update README.md (#452)
anitab-org/mentorship-backend: README.md

## Contact

> You can reach maintainers team @mentorship-team on [anitab-org Open Source Zulip](https://anitab-org.zulipchat.com) or @anitab-org/maintainers-mentorship-backend on GitHub. If you're interested in contributing to this project, join us there!

Reviewer: ```suggestion
You can reach the maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com). We have a [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system) stream dedicated to this project. If you're interested in contributing to this project, join us there!
```
Could you please change this ^^
